baselines/dom.generated.d.ts

```diff
@@ -5771,6 +5819,10 @@ interface GlobalEventHandlersEventMap {
     "transitionstart": TransitionEvent;
     "volumechange": Event;
     "waiting": Event;
+    "webkitanimationend": Event;
+    "webkitanimationiteration": Event;
+    "webkitanimationstart": Event;
+    "webkittransitionend": Event;
```

Comment:
this is probably fine, but it's slightly weird to me that webkit* event handlers are being added. Isn't that webkit-specific?

Reply:
Apparently these have been added to the HTML standard itself, see whatwg/html#5188. 🤷

Should I add them to `removedTypes.json`? I see there's already a couple of WebKit-specific removals in there.